src/components/event/EventFormContent.vue

```diff
@@ -147,7 +147,18 @@ export default class EventFormContent extends Vue {
       api.groups.getGroups(),
       api.groups.getMyGroups({ relation: GetMyGroupsRelationEnum.Belongs }),
     ])
-    this.allGroups = groups.filter(group => groupIds.includes(group.groupId))
+    const currentUser = this.$store.direct.state.me! // 現在のユーザー情報を取得しているので，nullではないとしてよい？
```

Comment:
```suggestion
    const currentUser = this.$store.direct.state.me
if(!currentUser) return
```
ここはこんな感じでearly returnしちゃうといいかと思います